packages/formation/sass/modules/_m-responsive-tables.scss

```diff
@@ -23,13 +23,7 @@ table.responsive {
   table.responsive {
     border: 0;
     thead {
-      border: none;
-      clip: rect(0, 0, 0, 0);
-      height: 1px;
-      margin: -1px;
-      overflow: hidden;
-      padding: 0;
-      width: 1px;
+      display: none;
```

Comment:
We want to hide this when `max-width` is representative of a mobile device's screen width.